data-loader/server.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask
+from app import controller
+import re
+import os
+import datetime
+
+facebook_data_root = "../data/derived/fib_results/facebook/"
+twitter_data_root = "../data/derived/fib_results/twitter/"
+PLATFORMS = ["Facebook", "Twitter"]
+LOAD_PAST_MONTH_DATA = ['2023_01']
+
+def load_current_data():
+    """
+    Load each platform and read the files.
+    """
+    for platform in PLATFORMS:
+        if platform == 'Facebook':
+            read_dir = facebook_data_root
+        else:
+            read_dir = twitter_data_root
+        all_year_month = get_available_year_month(read_dir)
+        current_year_month = datetime.datetime.now().strftime("%Y_%m")
+        selected_month = list(filter(lambda x: current_year_month in x, all_year_month))
+        if len(selected_month) == 1:
+            controller.read_files(read_dir, platform, selected_month[0])
+        else:
+            raise Exception("Error in finding data on", current_year_month)
+            logger.error("Error in finding data on {}", current_year_month)
+
+def load_past_data():
+    """
+    Load each platform and read the files.
+    """
+    for selected_month in LOAD_PAST_MONTH_DATA:
+        for platform in PLATFORMS:
+            if platform == 'Facebook':
+                read_dir = facebook_data_root
+            else:
+                read_dir = twitter_data_root
+            controller.read_files(read_dir, platform, selected_month)
+
+def get_available_year_month(root_dir):
```

Comment:
It doesn't look like this function is used in this script. Can we remove it?

Reply:
I removed the get_avaialble_year_month function. As there is no need to search for available months in the directory, I can call the current_year_month directory without searching for the entire directory